test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,93 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps as follows: 
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc. EC2 region, numbers of master nodes and worker nodes, and instance type can also be specified in playbook `vars`.
```

Comment:
"_can_ .. be specified" implies "can be unspecified". Is this true? If so, then the reader should be told what are the defaults. If not then say "must" rather than "can".

Reply:
ok, I see! I removed that sentence as I realize that it is not really needed.